### ISSUE TITLE
[Feature Request]: Open files >1000 columns faster

### ISSUE THREAD

**Issue by tomtomme** [jasp-stats/jasp-issues#3038]
### Description

importing many columns is still kinda slow, at least on linux. 

[data-4000column.csv](https://github.com/user-attachments/files/17980333/data-4000column.csv)

This file needed 100 seconds to import, while libreoffice calc just needed 3 seconds.

### Additional context
-------- Application Info --------
JASP Version: JASP 0.95 beta

**Comment by JorisGoosen** [comment]
Yes, well we just need to figure out which `sqlite` is being used on flatpak. Maybe its part of one of the runtime environments?

And then I guess we need to simply include it in org.jaspstats.JASP.json and add the buildflag @RensDofferhoff found before

**Comment by tomtomme** [comment]
Rens just implemented this for flatpak. Superfast!
I updated this request to be about the import speed. But this is certainly of low priority. Some minutes of importing are bearable as long as it works reliable.

**Comment by tomtomme** [comment]
even bigger file to test this here: 
https://github.com/jasp-stats/jasp-issues/issues/3282

**Comment by JorisGoosen** [comment]
This should be better now

**Comment by EJWagenmakers** [comment]
This should be much better in the upcoming version
